compiler/src/iree/compiler/Codegen/Common/VectorLayoutAnalysis.cpp

```diff
@@ -620,6 +647,27 @@ static void enforceLayoutToBroadcastOp(
   update(value, changed);
 }
 
+static void enforceLayoutToContractionOp(
+    vector::ContractionOp contraction,
+    ArrayRef<DistributionLayout *> operandLattices,
+    ArrayRef<const DistributionLayout *> resultLattices,
+    std::function<void(DistributionLayout *, ChangeResult)> update) {
+  // Contraction has only one vector result.
+  const DistributionLayout *result = resultLattices[0];
+  // Contraction has init value at position 2.
+  DistributionLayout *value = operandLattices[2];
+
+  // Cannot enforce layout if result is uninitialized.
+  if (result->isUninitialized()) {
+    return;
+  }
+
+  // True to resolve the init value with the result layout.
+  ChangeResult changed =
+      value->resolveWithPossibleConflict(result, getOpOperand(contraction, 2));
```

Comment:
Is it 2 because it is the `acc` of contractOp? If it is, isn't there a nicer helper function such as contraction.getAcc() or something?

Reply:
The 2 says that this is the 2nd **vector operand** of the contract, not the 2nd **operand** of the operation, which is why I had to pass it explicitly.

Also, getAcc returns a value. Here, we are using "OpOperand", which is a use of a value, not a value.